addons/sale_contract/models/sale_contract.py

```diff
@@ -328,6 +328,19 @@ def _get_total_invoiced(self, account):
             total_invoiced += account.timesheet_ca_invoiced
         return total_invoiced
 
+    def _get_total_overdue(self, cr, uid, ids, name, arg, context=None):
+        res = {}
+        ail_obj = self.pool['account.invoice.line']
```

Comment:
do not put postfix like _obj in variable. can you replace with 'AccountInvoiceLine' ?